deploy/sre-prometheus/100-alertmanager-silence-active.PrometheusRule.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       # upgrade process will generate a silence itself. (OSD-3426)
       # Use an average as without instance,pod, so that all AM pods are taken
       # into consideration
-      expr: avg without (instance,pod)(alertmanager_silences{state="active"}) > 0 unless (count(cluster_version{type="updating"}) > 0)
+      expr: avg without (instance,pod,endpoint,job,service,state,container)(alertmanager_silences{state="active"}) > 0 unless (count(cluster_version{type="updating"}) > 0 or sum by(namespace) (time() - kube_pod_created{namespace="openshift-monitoring",pod=~"osd-cluster-ready.*"} < 4500))
```

Comment:
Which is the rationale for that 4500 secs interval?

Reply:
4500 seconds is 1h15m, giving us a 15 minute window for the alerts to clear after the initial silence is created by this job.

Reply:
I would add/update a comment here, explaining the new  logic added for `osd-cluster-ready`